index.html

```diff
@@ -120,12 +123,72 @@
         --whiteBlankLink: #a9abe5;
         --mainYellow: #efab34;
         --mainPurple: #4a03d9;
-
-        
       }
       body {
-          margin: 0;
-        }
+        margin: 0;
+      }
+      .doenet-viewer {
+        font-family: 'Open Sans' !important;
+        color: var(--canvastext);
+      }
+      .doenet-viewer h1 {
+        font-size: 2em;
+        font-weight: bold;
+      }
+      .doenet-viewer h2 {
+        font-size: 1.5em;
+        font-weight: bold;
+        margin-block-start: 0.83em;
+        margin-block-end: 0.83em;
+      }
+      .doenet-viewer h3 {
+        font-size: 1.17em;
+        font-weight: bold;
+        margin-block-start: 1em;
+        margin-block-end: 1em;
+      }
+      .doenet-viewer h4 {
+        font-weight: bold;
+        margin-block-start: 1.33em;
+        margin-block-end: 1.33em;
+      }
+      .doenet-viewer h5 {
+        font-size: 0.83em;
+        font-weight: bold;
+        margin-block-start: 1.67em;
+        margin-block-end: 1.67em;
+      }
+      .doenet-viewer p {
+        margin-block-start: 1em;
+        margin-block-end: 1em;
+      }
+      .doenet-viewer input {
+        box-sizing: content-box;
+        padding: 2px 1px;
+      }
```

Comment:
For future reference, you can next all of these within the first .doenet-viewer selector for a little less repetition.